### DISCUSSION TITLE
115小幸运好久没推送新版本了！

### DISCUSSION THREAD

**Discussion by kaixinxiaowangzi666** [AlistGo/alist#5594] [General]
一直都是旧版本

**Comment by user1121114685** [comment]
欢迎来到小幸运2的世界，这是一款集成了Alist全功能的网盘播放器，我们采用libmpv作为播放核心，致力于为您提供最佳的观影体验。
我们的愿景是让小幸运2成为您的私人影院，让您能像在爱奇艺、腾讯视频、优酷视频等平台一样方便地观看视频。
我们坚持简单、易用和美观的设计理念，为您提供一个纯净的观影环境。因此，小幸运2没有任何内置广告，也不会收集您的个人信息。
我们承诺，小幸运2将永远免费，让每一位用户都能随时随地享受到我们的服务。
然而，由于我们坚持免费且无广告的原则，小幸运2没有任何商业推广渠道。
因此，我们需要您的帮助来让更多的人知道并使用小幸运2。
如果您喜欢我们的产品，请分享给您的朋友和家人，让我们一起点亮星光，让小幸运2的光芒照亮更多的人。

一键添加多个网盘
在'管理' => '+'中，您可以快速添加115网盘、天翼云盘、阿里云盘等多种网盘。特别是对于115网盘，我们提供了多达5种登录方式，确保您的使用体验始终流畅。

无需Alist密码
无论您是添加网盘，还是管理Alist，都无需输入密码，直接进行操作。这大大方便了对密码记忆有困难的用户。

请求速度保护
为了防止用户在使用过程中触发网盘的请求保护功能，导致网盘长时间无法访问，小幸运2特别引入了请求速度保护机制，让您可以有序、无限制地发送请求，顺利完成任务。

拥有电影海报
只要您的文件夹内包含.jpg、.nfo和视频三个元素，就能自动触发海报功能。精美的海报将为您的观影体验增添色彩。

源码播放可选字幕音轨
小幸运2的在线播放功能不仅保证了画质，还提供了音轨（多种语言选择）和字幕（多种字幕支持）的选择，让您的观影体验更加丰富多彩。

网盘的全局搜索与过滤
小幸运2已经支持阿里云盘和115网盘的全局搜索，并且提供了文件夹过滤功能，让您能够一步找到所需文件，节省您的宝贵时间。

强制刷新与文件夹位置记忆
当您更新了网盘文件后，无需重启Alist，小幸运2提供了一键刷新文件夹的功能。同时，当您进入文件夹后，系统会自动保存当前进度，您返回时无需从头开始寻找。

安卓下载地址: https://wwi.lanzoup.com/iM5uc1v7xvdc

Win10 X64下载地址: https://wwi.lanzoup.com/iKoix1v7xwza

Ios下载地址: 敬请期待

Ipd下载地址: 敬请期待

MAC M1 芯片下载地址: 敬请期待

MAC X64 芯片下载地址: 敬请期待

**Reply by kaixinxiaowangzi666** [reply]
谢谢
